engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1595,6 +1591,23 @@ final class SystemColor {
   ///
   ///   * [isSupported], which returns whether a specific color is supported.
   static bool get platformProvidesSystemColors => false;
+}
+
+/// A palette of system colors specified in the operating system for a given [brightness].
+///
+/// The getters in this class, such as [accentColor] and [buttonText], provide standard system
+/// colors defined by the [W3C CSS specification](https://drafts.csswg.org/css-color/#css-system-colors).
+final class SystemColorPalette {
+  SystemColorPalette._(this.brightness);
+
+  /// A palette of system colors for light mode.
```

Comment:
will the value for light or dark change based on the light/dark setting of the browser?

Reply:
No. The user can get both light and dark system colors, regardless of system brightness settings.